### ISSUE TITLE
Breaks `:mksession`

### ISSUE THREAD

**Issue by Integralist** [dstein64/nvim-scrollview#71]
👋🏻 

I've just spent some time debugging an issue where when I use `:mksession!` and try to restore the session via `nvim -S` flag the restored session layout is completely broken.

Turns out the problem goes away if I disable this plugin.

**Comment by dstein64** [comment]
Thanks for reporting the issue @Integralist!

I've reproduced the issue outside the context of `nvim-scrollview` and reported the issue to the Neovim project.
https://github.com/neovim/neovim/issues/18432.

The following `MkSession` command can work around the issue (assuming there are not also other non-scrollbar floating windows that need to be closed):

```vim
command -bang -nargs=? MkSession
      \ silent! ScrollViewDisable
      \ | mksession<bang> <args>
      \ | silent! ScrollViewEnable
```

**Comment by dstein64** [comment]
This is addressed in [Neovim #18635](https://github.com/neovim/neovim/pull/18635). The fix is included in the [nightly release](https://github.com/neovim/neovim/releases/tag/nightly) and will presumably be included in the future [v0.7.1](https://github.com/neovim/neovim/milestone/27) and/or [v0.8.0](https://github.com/neovim/neovim/milestone/28) release.
